build.gradle

```diff
@@ -1,46 +1,53 @@
-import se.bjurr.gitchangelog.plugin.gradle.GitChangelogTask
```

Comment:
This file was recreated by copying the base forge `build.gradle` and reintroducing function calls until things worked again. I assume I missed some important steps